liquid-staking/src/basics/constants.rs

```diff
@@ -0,0 +1,8 @@
+pub const DEFAULT_GAS_TO_CLAIM_REWARDS: u64 = 6_000_000;
+pub const MIN_GAS_FOR_ASYNC_CALL: u64 = 12_000_000;
+pub const MIN_GAS_FOR_CALLBACK: u64 = 12_000_000;
+pub const MIN_EGLD_TO_DELEGATE: u64 = 1_000_000_000_000_000_000;
+pub const RECOMPUTE_BLOCK_OFFSET: u64 = 10;
+pub const MINIMUM_LIQUIDITY: u64 = 1_000;
+pub const DEFAULT_MIN_GAS_TO_SAVE_PROGRESS: u64 = 30_000_000;
+pub const MIN_GAS_FINISH_EXEC: u64 = 20_000_000;
```

Comment:
I don't really like having so many `u64` types that mean very different things. At the very least, define some types:
```
pub type GasLimit = u64;
pub type Blocks = u64;
```